en/computer_vision/obstacle_avoidance.md

```diff
@@ -71,43 +99,26 @@ The fields are set as shown:
   - `acc_x[0]`, `acc_y[0]`, `acc_z[0]`: x-y-z NED acceleration setpoint
   - `pos_yaw[0]`: Current yaw angle
   - `vel_yaw[0]`: NaN
+  - `command[0]`: NaN
```

Comment:
@mrivi Is the command 0 for the current position in missions an NaN? Are the other two correct?

Reply:
This changed. We are sending the type adapted current triplet in Point0. This is the information we get out of FlightTaskAutoMapper.
FlightTaskAuto gets the triplets from navigator and reprojects them in local coordinates. FlightTaskAutoMapper takes into consideration the type of waypoint e.g. Land, Loiter, Takeoff... and sets velocity components if needed. In the case of a Land waypoint, the z component of the position is set to NAN and the z component of the velocity is set. In this way, xy is position controlled and z is velocity controlled.

Reply:
I think I'm going to need a bit more information to understand this. Point0 is the current information about the vehicle right? 
Why is it not useful to send the z component of the position? I can see why you might say "I want you to velocity control the target point" but I would have thought that would be something in point1.

Reply:
No, it isn't anymore. It's the current waypoint in local coordinated changed by FLightTaskAutoMapper based on the type of the waypoint.

Reply:
Then what are points 1 and 2?